packages/@aws-cdk/aws-synthetics/lib/canary.ts

```diff
@@ -384,6 +424,19 @@ export class Canary extends cdk.Resource {
     };
   }
 
+  /**
+   * Retruns a runConfig object
+   */
+  private createRunConfig(props:CanaryProps): CfnCanary.RunConfigProperty {
+    const DEFAULT_CANARY_TIMEOUT_IN_SECONDS = 900;
```

Comment:
Since TimeoutInSeconds is optional, not specifying a default here would allow the service to use the defaults that it sets.

Reply:
Is not optional (https://github.com/aws/aws-cdk/issues/9300). But right I'll implement the same logic as stated in the doc: "If you omit this field, the frequency of the canary is used as this value, up to a maximum of 900 seconds." (https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-synthetics-canary-runconfig.html#cfn-synthetics-canary-runconfig-timeoutinseconds)